test/dotnet-migrate.Tests/GivenThatIWantToMigrateSolutions.cs

```diff
@@ -14,6 +14,83 @@ namespace Microsoft.DotNet.Migration.Tests
 {
     public class GivenThatIWantToMigrateSolutions : TestBase
     {
+        private const string ExpectedSlnFileAfterRemovingAllSolutionItems = @"
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Can we do something else other than this? I would prefer that in the tests we simply cracked the file open and checked only what the test cares about, instead of having this text blob be a part of the tests.